smartfs.lua

```diff
@@ -595,29 +604,23 @@ smartfs.element("checkbox",{
 
 smartfs.element("list",{
         build = function(self)
-                local listformspec = "textlist["..
-                                     self.data.pos.x..","..self.data.pos.y..
-                                     ";"..
-                                     self.data.size.w..","..self.data.size.h..
-                                     ";"..
-                                     self.data.name..
-                                     ";"
-
-                --loop through the list's items and add them to the formspec
 		if not self.data.items then
-			self.data.items = {" "}
+			self.data.items = {}
 		end
-                for i,value in ipairs(self.data.items) do
-                    listformspec = listformspec..value..","
-                end
-                listformspec = string.sub(listformspec, 0, -2) --removes extra ,
-                --close out the list items section
-                listformspec = listformspec..";"
-
-                --TODO support selected idx and transparency
+		print(dump(self.data.items))
```

Comment:
haha i thought i removed it, its upto you i can removeit before merge or you can do it after. what ever is best for you :package :P